### ISSUE TITLE
Theme.Sherlock.Light tab bar background un-light

### ISSUE THREAD

**Issue by thorinside** [JakeWharton/ActionBarSherlock#108]
It seems, with the latest ABS, the Light theme no longer has a light background on the tab bar. It shows up as a very dark grey, with black text on top. I've been trying to figure out what to set to change this, and I'll likely find it eventually, but I'm sure you guys could solve this super fast. Any clues?

**Comment by JakeWharton** [comment]
I'll release a new version **very** soon to fix this but for now:

``` xml
<style name="Theme.MyApplication" parent="Theme.Sherlock">
    <!-- your customizations here -->

    <item name="actionBarTabBarStyle">@style/Widget.MyApplication.ActionBar.TabBar</item>
</style>

<style name="Widget.MyApplication.ActionBar.TabBar" parent="Widget.Sherlock.ActionBar.TabBar">
    <item name="android:background">@color/actionbar_tabbar_background</item>
</style>

<color name="actionbar_tabbar_background">0xFFFF0000</color>
```

This will change it to bright red. Tweak as necessary.

**Comment by thorinside** [comment]
Thanks, I'll give that a whirl.

**Comment by thorinside** [comment]
Sorry, your solution wouldn't work even after I made the following substitutions:

attr -> item and 0x -> # 

I needed to remove line 88 from abs__action_bar.xml to get that working.

**Comment by JakeWharton** [comment]
I had hoped the style would override the specified value. I'll try to release a new version tonight then which fixes this.

**Comment by JakeWharton** [comment]
Fix released as v3.4.2.

**Comment by thorinside** [comment]
Thanks!

**Comment by bhucker** [comment]
How would I change the color of the bars at the bottom of the tabs that show which tab is selected?

**Comment by JakeWharton** [comment]
The tab backgrounds are a state-list drawable. You provide different
9-patches for the selected state.
On May 30, 2012 12:19 PM, "Bob Hucker" <
reply@reply.github.com>
wrote:

**Comment by bhucker** [comment]
Thank you, but I have tried modifying the images named, for example,
abs__tab_selected_holo.9.png. I changed all the ones with "selected" in the
name within drawable-hdpi, drawable-mdpi and drawable-xhdpi. It seemed to
have no effect. Are these the right images?

I was looking at the drawables mentioned in abs__tab_indicator_holo.xml.

--Bob

On Wed, May 30, 2012 at 12:22 PM, Jake Wharton <
reply@reply.github.com
